docs/src/pages/[platform]/getting-started/usage/usage.react.mdx

```diff
@@ -0,0 +1,55 @@
+import { Tabs, TabItem } from '@aws-amplify/ui-react';
+import { Example, ExampleCode } from '@/components/Example';
+
+## Import styles and components
+
+Load the default styling by importing our CSS:
+
+```jsx
+import { View } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css'; // default theme
+
+export default function App() {
+  return <View>Your app here</View>;
+}
+```
+
+## Custom Theme
+
+To add a [custom theme](/theming), wrap your app in the `AmplifyProvider` component and include the default styling.
```

Comment:
I guess I'm confused here because this example doesn't actually demonstrate how to use a custom theme (by passing a Theme object). The default styling will get applied by importing `@aws-amplify/ui-react/styles.css`, even without using `AmplifyProvider`. For this section, perhaps we can simply include a link to the [Theming page](https://ui.docs.amplify.aws/react/theming)?

Reply:
It might be an ok interim fix to just remove this section? I agree it doesn't really provide useful info for starting a custom theme, even as is on the live site currently. In addition this page might change quite a bit when we work on the actual Usage page task. Additionally, we have a task to add "Next steps" links that could help with adding nice link outs to other docs pages.

Reply:
how many times can I say the word "additionally" lol

Reply:
+1 that this section doesn't make much sense. We can just remove it for now and address this in the usage ticket.

Reply:
Thanks! Removed that text for now in latest commits.